### ISSUE TITLE
Team project adding new members issue

### ISSUE THREAD

**Issue by kamushken** [stackblitz/core#1285]
Guys, I've purchased "team of two" access, but no idea on how to manage my team to add more users. My profile page looks very scary (screenshot https://imgur.com/DVUyhzR) and clicking on Add members link leads to nowhere. Chrome/Edge tested. I am very unsatisfied how my paid journey has began. And no idea how to contact you via website.

**Comment by riapacheco** [comment]
I know this is from a while ago but I also signed up for two members and had the same frustrating experience. Turns out there’s a hidden input field to the right of `Username` on the manage members page. I pasted my team member’s GitHub username and it looks like it worked; but following up with that team member now. They likely just have to log into Stackblitz though. Hope this helps (or at least for anyone googling this).

![Screen Shot 2020-11-03 at 2 52 53 PM](https://user-images.githubusercontent.com/53801436/98044658-75bcf980-1de4-11eb-96ae-3c4b170dbb2b.png)
